collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -3588,6 +3588,7 @@ void update_cgroup_charts(int update_every) {
                                 rrddim_add(cg->st_cpu_limit, "used", NULL, 1, system_hz, RRD_ALGORITHM_ABSOLUTE);
                             else
                                 rrddim_add(cg->st_cpu_limit, "used", NULL, 1, 1000000, RRD_ALGORITHM_ABSOLUTE);
+                            cg->prev_cpu_usage = (calculated_number)(cg->cpuacct_stat.user + cg->cpuacct_stat.system) * 100;
```

Comment:
a suggestion, adding a new line after if-else block is easy on the eyes especially when if-else block is missing curly braces in C

Reply:
ops bad timing 😅 I agree with the suggestion tho

Reply:
> when if-else block is missing curly braces in C

for me it is hard to read too, but I follow the existing code style 😞